src/test/lib/scheduler/operator_task_test.cpp

```diff
@@ -18,13 +18,28 @@
 #include "scheduler/node_queue_scheduler.hpp"
 #include "scheduler/operator_task.hpp"
 
+namespace {
+
+using namespace hyrise;  // NOLINT(build/namespaces)
```

Comment:
Just a pick here: we could actually nest our anonymous namespaces inside the `hyrise` namespace and avoid `using namespace hyrise;` everywhere

Reply:
Hmm, nesting namespaces sounds a bit more confusing than this single line. Not sure I am in favor.

Reply:
I don't mean that the act of nesting in confusing but it already find it often hard to immediately see if a function is in the anon namespace of not. With nesting this sounds even harder.

Reply:
I have no strong opinion here, just came to my mind. At least, nesting would save us a lot of `NOLINT`s